openadapt/app/tray.py

```diff
@@ -64,12 +66,18 @@ def __init__(self) -> None:
         self.replay_menu = self.menu.addMenu("Replay")
         self.populate_menu(self.replay_menu, self._replay, "replay")
 
-        self.app_action = QAction("Show App")
-        self.app_action.triggered.connect(self.show_app)
```

Comment:
I think we should not remove this until all of the functionality in the legacy app has been implemented in the dashboard. How about a `TODO: remove` for now?

Reply:
Right, makes sense